content/en/blogs/Will Cloud Native WebAssembly Replace Docker_.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Will Cloud Native WebAssembly Replace Docker?'
+keywords: Cloud Native, WebAssembly, Kubernetes
+description: Given the advantages of WebAssembly, can it replace Docker?
+tag: 'Cloud Native, WebAssembly, Kubernetes'
+createTime: '2021-07-28'
+author: 'Feynman, Jianna Liu, Yicen Xie'
+snapshot: 'https://raw.githubusercontent.com/VeraXIE1997/test/main/Will%20Cloud%20Native%20Replace%20Docker/image/image-20210721141953114%E7%9A%84%E5%89%AF%E6%9C%AC.png'
+---
+
+# Will Cloud Native WebAssembly Replace Docker?
```

Comment:
This title is duplicated with the top one, preview it [here](https://deploy-preview-1843--kubesphere-v3-1.netlify.app/blogs/will-cloud-native-webassembly-replace-docker_/).